### DISCUSSION TITLE
Enhancement idea: use a JS parser for Computed Data instead of Proxies

### DISCUSSION THREAD

**Discussion by zachleat** [11ty/eleventy#4074] [Enhancement Queue]
I don’t know the performance differences here but I’m logging the idea for later.

https://www.npmjs.com/package/acorn

If you think this is not a good idea, please let me know!

**Comment by zachleat** [comment]
> This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

> View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!

**Comment by Ryuno-Ki** [comment]
When I mentioned that (resp. Esprima), I was recommended to try [meriyah](https://github.com/meriyah/meriyah):
https://twitter.com/ErikArvidsson/status/1336012973232439296

// CC @arv

**Comment by zachleat** [comment]
Related to #4065
